avaframe/runScripts/runRenaming.py

```diff
@@ -0,0 +1,31 @@
+"""
+    Run script for getting rename dataframe
+"""
+
+# Load modules
+import logging
+import pathlib
+
+# Local imports
+#  from avaframe.in3Utils import cfgUtils
+from avaframe.in3Utils import cfgHandling
+from avaframe.in3Utils import logUtils
+
+
+# log file name; leave empty to use default runLog.log
+logName = 'runRenaming'
+
+#  Load general configuration file
+#  cfgMain = cfgUtils.getGeneralConfig()
+
+avaDir = 'data/avaAlr'
```

Comment:
is this wanted? I would uncomment the cfgMain and remove the hard coded avaDir no?